installDependencies.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Setup
+# Exit the script if any command fails
+set -e
+# Update apt
+sudo apt update
+# GCC 8
+sudo apt install gcc-8 g++-8
```

Comment:
In https://github.com/SoPra-Team-17/LibCommon/pull/4 sieht das so aus als ob GCC-8 schon installiert wäre, aber halt nicht standardmäßig benutzt wird.
Ich würde das hier einfach mal rausnehmen aber die Umgebungsvariablen behalten, damit die deps mit GCC8 gebaut werden